packages/drivers/odsp-driver/src/test/odspDriverResolverTest.spec.ts

```diff
@@ -132,11 +131,7 @@ describe("Odsp Driver Resolver", () => {
 
     it("Should resolve url with a IFluidPackage in the codeDetails package", async () => {
         const resolvedUrl = await resolver.resolve(request);
-        const fluidPackage: IFluidPackage = {
-            name: packageName,
-            fluid: {},
-        };
-        const codeDetails = { package: fluidPackage };
+        const codeDetails = { name: packageName };
```

Comment:
These tests should be fully restored to their original state (with the PR as-is, the three are now all testing the same `IContainerPackageInfo` codepath, leaving the `IFluidCodeDetails` approach untested)